cfme/cloud/provider.py

```diff
@@ -125,6 +125,14 @@ def __init__(self, name=None, credentials=None, zone=None, key=None):
     def _form_mapping(self, create=None, **kwargs):
         return {'name_text': kwargs.get('name')}
 
+    @property
+    def is_cloud(self):
+        return True
+
+    @property
+    def is_infra(self):
+        return False
```

Comment:
Can these things be made more simple? Like a class attribute or something like that?